crates/uv-resolver/src/flat_index.rs

```diff
@@ -226,3 +226,9 @@ impl From<FlatDistributions> for BTreeMap<Version, PrioritizedDist> {
         distributions.0
     }
 }
+
+impl From<BTreeMap<Version, PrioritizedDist>> for FlatDistributions {
```

Comment:
```suggestion
/// For external users.
impl From<BTreeMap<Version, PrioritizedDist>> for FlatDistributions {
```

Making sure this doesn't get removed as unused.